tests/performance_manager/test_performance_manager.py

```diff
@@ -85,7 +85,10 @@ def test_files() -> List[str]:
         for file in files:
             if "DS_Store" in file:
                 continue
-            paths.append(os.path.join(root, file))
+            path = os.path.join(root, file)
+            if "year=2024" in path:
+                continue
+            paths.append(path)
```

Comment:
temp patch to stop this test from consuming new data.

ideally, we would have one set of vp files that work for both tests.